org.flatpak.demo.json

```diff
@@ -0,0 +1,35 @@
+{
+    "app-id": "org.flatpak.demo",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.10",
+    "sdk": "org.kde.Sdk",
+    "command": "flatpak-demo",
+    "rename-icon": "flatpak",
```

Comment:
Icons should probably have the correct name upstream?

Reply:
This is intended to demonstrate use of this command.